generator/generator_tests/osm_type_test.cpp

```diff
@@ -936,3 +936,622 @@ UNIT_CLASS_TEST(TestWithClassificator, OsmType_DoNotMergeTags)
     TEST(!params.IsTypeExist(GetType({"shop", "photo"})), (params));
   }
 }
+
+UNIT_CLASS_TEST(TestWithClassificator, OsmType_SimpleTypesSmoke)
+{
+  Tags const simpleTypes = {
```

Comment:
как мы этот список получили?

Reply:
это все типы из mapcss-mapping.csv?

Reply:
Это все короткие правила из mapcss (все записи длины 3, у которых в последнем столбце не стоит x)

Reply:
мб добавить в mapcss-mapping.csv последний строкой:
"# If you are going to add new type, add it to generator/generator_tests/osm_type_test.cpp in case of short rule ... blah blah blah"
?

Reply:
ну и при парсинге игнорировать строки, начинающиеся с `#`

Reply:
не очень хочется ввязываться в изменение всех парсилок mapcss-maping, тем более часть из них в сабмодулях